root/static/scripts/common/utility/entityHref.js

```diff
@@ -19,7 +19,7 @@ type LinkableEntity =
   | {+entityType: 'editor', +name: string, ...}
   | {+entityType: 'isrc', +isrc: string, ...}
   | {+entityType: 'iswc', +iswc: string, ...}
-  | {+entityType: CoreEntityTypeT, +gid: string, ...}
+  | {+entityType: RelatableEntityTypeT, +gid: string, ...}
```

Comment:
@reosarevok wrote in https://github.com/metabrainz/musicbrainz-server/commit/14988ae5210b93d1e841f121a215f35d620af210#r103558464:
> Genre for example was linkable and a "CoreEntity" before it supported relationships. Although we might not implement things like that in the future, and I guess it would be ok to just have any other entities like that be added here separately like `collection` and `link_type`.

@bitmap answered:
> Right. I'm just asserting that all relatable entities are linkable, not that they comprise all or most linkable entities. (But I also wouldn't have considered genre a core entity until they supported relationships, which I guess helps prove the vagueness of the term.)

That’s about commit https://github.com/metabrainz/musicbrainz-server/pull/2896/commits/63fc9201b9fc417861cb2bb82accd3b419ab831a.

Reply:
It has been further discussed on IRC and agreed it was an non-issue as it can be extended using union type if needed when a non-relatable entity type has to use this. That was already the case in many components using `| CollectionT` for example.